pkg/crc/api/helpers.go

```diff
@@ -40,6 +40,11 @@ func (c *context) String(code int, r string) error {
 	return err
 }
 
+func (c *context) Code(code int) error {
+	c.code = code
+	return nil
+}
+
```

Comment:
This helper function is always going to send `nil` error and what it actually doing is just assigning the `c.code = code` we can just make `code` exportable from the struct and have use it instead creating a helper function just to do that or better name this helper to `SetCode`?

Reply:
it makes no sense to have an error that always returns nil; unexpected side-effect.

Reply:
> This helper function is always going to send `nil` error and what it actually doing is just assigning the `c.code = code` we can just make `code` exportable from the struct and have use it instead creating a helper function just to do that or better name this helper to `SetCode`?

Yeah, `Code` is not a good name, i went with it to be consistent since other methods are named like this `c.JSON`, `c.String` to write json and text as response. 

about always returning `nil`, if we don't return an error from this function we have to return a nil from the handlers.. doing it in this function allows us to write `return c.Code(httpOK)` in the handlers.